### DISCUSSION TITLE
microsoft-office-po-staremu-klasyczne-menu-office/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#126] [General]
# microsoft-office-po-staremu-klasyczne-menu-office/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/microsoft-office-po-staremu-klasyczne-menu-office/?utm_medium=rss&utm_source=rss&utm_campaign=rss

**Comment by spexpl** [comment]
OT: Czy tylko mi zwinięte pole "Komentarze i Reakcje" powoduje odruch, iż tu się nie da komentować (iż komentarze zostały zkliwidowane/wyłączone)?

**Reply by idarek** [reply]
Testowałem różne ustawienia. Jak ktoś chce skomentować, to skomentuję. Szkoda wczytywać skrypt komentowania za każdym razem.

Pomyślę aby to nazwać inaczej? Aby zachęcić do kliknięcia :) 

U mnie jak komentarze są wyłączone w danym wpisie to tej sekcji w ogóle nie ma :)
